insights/parsers/ovs_ofctl_dump_flows.py

```diff
@@ -0,0 +1,85 @@
+"""
+OVSofctlDumpFlows - command ``/usr/bin/ovs-ofctl dump-flows <bridge-name>``
+===========================================================================
+
+This module provides class ``OVSofctlDumpFlows`` to parse the
+output of command ``/usr/bin/ovs-ofctl dump-flows <bridge-name>``.
+"""
+
+
+from insights import CommandParser, parser
+from insights.parsers import SkipException
+from insights.specs import Specs
+from ..parsers import split_kv_pairs
```

Comment:
Please do not mix `full path` importing and `relative` importing in one single file.

L13 can be merged to L11

Reply:
Done